sdc/tests/tests_perf/test_perf_numpy.py

```diff
@@ -85,6 +85,24 @@ def _test_case(self, cases, name, total_data_length, data_num=1, input_data=test
         CE(type_='Numba', code='data.astype(np.int64)', jitted=True),
         CE(type_='SDC', code='sdc.functions.numpy_like.astype(data, np.int64)', jitted=True),
     ], usecase_params='data'),
+    TC(name='nanargmin', size=[10 ** 7], call_expr=[
```

Comment:
Why did not you add cases for Numba for `np.nanargmin` and `np.nanargmax`?

Reply:
Because its not exist. Numba dont have this methods